tgui/packages/tgui/interfaces/Zubbers_Storyteller.tsx

```diff
@@ -0,0 +1,467 @@
+      <Table>
+        <Table.Row bold>
+          <Table.Cell width={TRACK_DATA_TRACK_WIDTH}>Track</Table.Cell>
+          <Table.Cell width={TRACK_DATA_POINT_WIDTH}>Points</Table.Cell>
+          <Table.Cell width={TRACK_DATA_NEXT_WIDTH} textAlign="center">
+            Next event
+          </Table.Cell>
+          <Table.Cell width={TRACK_DATA_FORCED_WIDTH}>
+            Next forced event
+          </Table.Cell>
+          <Table.Cell width={TRACK_DATA_ACTIONS_WIDTH}>Actions</Table.Cell>
+        </Table.Row>
+        <Stack.Divider />
+        {Object.entries(tracks_data).map(([track, track_data]) => {
+          const max_points = track_data.max;
+          const current_points = track_data.current;
+          const forced = track_data.forced ? track_data.forced : 0;
+          return (
+            <Table.Row key={track}>
+              <Table.Cell>
+                <Button
+                  tooltip="Edit points"
+                  width="100%"
+                  textAlign="center"
+                  onClick={() =>
+                    act('track_action', { action: 'set_pnts', track: track })
+                  }
+                >
+                  {track}
+                </Button>
+              </Table.Cell>
+              <Table.Cell>
+                <ProgressBar
+                  value={current_points}
+                  maxValue={max_points}
+                  ranges={{
+                    good: [-Infinity, max_points],
+                    average: [max_points, Infinity],
+                  }}
+                >
+                  {current_points + ' / ' + max_points}
+                  {' (' +
+                    Math.floor((current_points * 100) / max_points) +
+                    '%) '}
+                </ProgressBar>
+              </Table.Cell>
+              <Table.Cell textAlign="center">
+                {storyteller_halt ? 'N/A' : '~' + track_data['next'] + 'min'}
+              </Table.Cell>
+              <Table.Cell>{forced ? forced.name : ''}</Table.Cell>
+              <Table.Cell>
+                <Button
+                  onClick={() =>
+                    act('track_action', { action: 'force_next', track: track })
+                  }
+                >
+                  Next Event
+                </Button>
+              </Table.Cell>
+            </Table.Row>
+          );
+        })}
+      </Table>
+    </Section>
+  );
+};
+
+export const ZubbersStorytellerScheduledData = (props) => {
+  const { act, data } = useBackend<Storyteller_Data>();
+  const { scheduled_data } = data;
+  return (
+    <Section title="Scheduled events">
+      <Table>
+        <Table.Row bold>
+          <Table.Cell>Name</Table.Cell>
+          <Table.Cell>Track</Table.Cell>
+          <Table.Cell>Time</Table.Cell>
+          <Table.Cell>Actions</Table.Cell>
+        </Table.Row>
+        {Object.entries(scheduled_data).map(([event_name, event_data]) => {
+          const time = event_data['time'];
```

Comment:
```suggestion
          const timeNum = Number(time)?.toFixed(1);
```

Reply:
Use a actual number and lock it to one decimal so it doesn't constantly move around the place, or remove decimal entirely